tfhe/src/core_crypto/experimental/entities/pseudo_ggsw_ciphertext.rs

```diff
@@ -56,6 +58,146 @@ pub fn pseudo_ggsw_level_matrix_size(
     glwe_size_in.to_glwe_dimension().0 * glwe_size_out.0 * polynomial_size.0
 }
 
+/// Return the number of mask samples used during encryption of a [`PseudoGgswCiphertext`] given an
+/// input and output [`GlweSize`], [`PolynomialSize`] and [`DecompositionLevelCount`].
+pub fn pseudo_ggsw_ciphertext_encryption_mask_sample_count(
+    input_glwe_size: GlweSize,
+    output_glwe_size: GlweSize,
+    polynomial_size: PolynomialSize,
+    decomp_level_count: DecompositionLevelCount,
+) -> EncryptionMaskSampleCount {
+    EncryptionMaskSampleCount(
+        decomp_level_count.0
+            * pseudo_ggsw_level_matrix_encryption_mask_sample_count(
+                input_glwe_size,
+                output_glwe_size,
+                polynomial_size,
+            )
+            .0,
+    )
+}
+
+/// Return the number of mask samples used during encryption of a [`PseudoGgswLevelMatrix`] given an
+/// input and output [`GlweSize`], and [`PolynomialSize`].
+pub fn pseudo_ggsw_level_matrix_encryption_mask_sample_count(
+    input_glwe_size: GlweSize,
+    output_glwe_size: GlweSize,
+    polynomial_size: PolynomialSize,
+) -> EncryptionMaskSampleCount {
+    EncryptionMaskSampleCount(
+        input_glwe_size.0
+            * glwe_ciphertext_encryption_mask_sample_count(
+                output_glwe_size.to_glwe_dimension(),
+                polynomial_size,
+            )
+            .0,
+    )
+}
+
+/// Return the number of noise samples used during encryption of a [`PseudoGgswCiphertext`] given an
+/// input [`GlweSize`], [`PolynomialSize`] and [`DecompositionLevelCount`].
+pub fn pseudo_ggsw_ciphertext_encryption_noise_sample_count(
+    input_glwe_size: GlweSize,
+    polynomial_size: PolynomialSize,
+    decomp_level_count: DecompositionLevelCount,
+) -> EncryptionNoiseSampleCount {
+    EncryptionNoiseSampleCount(
+        decomp_level_count.0
+            * pseudo_ggsw_level_matrix_encryption_noise_sample_count(
+                input_glwe_size,
+                polynomial_size,
+            )
+            .0,
+    )
```

Comment:
Maybe impl trait to do multiplication `usize * EncryptionNoiseSampleCount`

Reply:
that's a good idea indeed, thanks, I'll do the same for the Mask thing I think